### PR TITLE
Post contain comments url

### PR DESCRIPTION
**Pull request by HailanXyouknow**
Post object has a field 'comments' which returns the comments URL.

### CODE REVIEW

File: server/main/utils.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 from enum import Enum
 from django.conf import settings
 
-HOST = "https://konnection-client.herokuapp.com"
+HOST = "https://konnection-server.herokuapp.com"
```
Comment (IanSeng) [inline server/main/utils.py:9]: ✨ thank you

**Review by IanSeng — APPROVED** [review]
LGTM ✨

File: server/posts/views.py [hunk 2]
```diff
@@ -195,7 +195,6 @@ def post(self, request, *args, **kwargs):
             return Response({'error': 'Post not found!'},
                             status=status.HTTP_404_NOT_FOUND)
         post_data = PostSerializer(a_post).data
-        print(post_data)
```
Comment (keanweng97) [inline server/posts/views.py:198]: thanks for catching this! my bad 😅

**Review by keanweng97 — APPROVED** [review]
LGTM!!! ✨✨✨

**Pull request merged**